src/button.tsx

```diff
@@ -21,19 +21,20 @@ export const BUTTON_MIN_WIDTH = 136;
 const transitionTiming = '0.3s cubic-bezier(0.77, 0, 0.175, 1)';
 
 const BORDER_PX = 1.5;
-const SPINNER_MARGIN_PX = 8;
+const ICON_MARGIN_PX = 8;
 const X_PADDING_PX = 16 - BORDER_PX;
 const Y_PADDING_PX = 12 - BORDER_PX;
 const X_SMALL_PADDING_PX = 12 - BORDER_PX;
 const Y_SMALL_PADDING_PX = 6 - BORDER_PX;
+const SMALL_ICON_SIZE = 20;
+const ICON_SIZE = 24;
 
```

Comment:
something happens in playroom snippet
https://ibit.ly/7ygh

Reply:
weird... it works fine in storybook. I'll take a look